templates/zerver/emails/digest.html

```diff
@@ -2,6 +2,72 @@
 Mail sent to a user who hasn't logged in for 24 hours.
 #}
 
+<span style="display:none">
```

Comment:
Added a `{% block preheader %}` to the `email_base` template (see https://github.com/zulip/zulip/pull/5547)